pkg/odo/util/completion/completionhandlers.go

```diff
@@ -68,6 +69,11 @@ var StorageDeleteCompletionHandler = func(cmd *cobra.Command, args parsedArgs, c
 	if err != nil {
 		return completions
 	}
+	devObj, err := devfile.ParseAndValidateFromFile(envInfo.GetDevfilePath())
+	if err != nil {
+		return completions
+	}
+	envInfo.SetDevfileObj(devObj)
```

Comment:
There was a bug when doing `odo storage delete <TAB>` because this function was trying to access the content of the devfile data (into `envInfo.ListStorage()`), but the devfile was not loaded at this time.

Reply:
Beautiful! Good thing that you caught the bug!